src/Web/WebClient.php

```diff
@@ -395,19 +395,19 @@ protected function detectEngine($userAgent)
             $this->engine = self::BLINK;
         } elseif (\stripos($userAgent, 'Chrome') !== false) {
             $result  = \explode('/', \stristr($userAgent, 'Chrome'));
-            $version = \explode(' ', $result[1]);
+            $version = isset($result[1]) ? \explode(' ', $result[1]) : false;
 
-            if ($version[0] >= 28) {
+            if ($version === false || $version[0] >= 28) {
```

Comment:
Maybe it should be the same as AppleWebKit?. I assume `self::WEBKIT` be the fallback value.
`$version !== false &&`

Reply:
I would assume an actual version if no version is given. "Chrome Privacy Preserving Prefetch Proxy" is a real user agent used for the https://developer.chrome.com/blog/private-prefetch-proxy feature of Google Chrome.

Reply:
I've added a unit test, and it fails with a reason. The detectBrowser would need to be adjusted to not detect a version="Privacy " when the user agent is "Chrome Privacy Preserving Prefetch Proxy"

Reply:
Changed to draft status until I have checked further.

Reply:
I thin it is OK to use `if ($version === false || $version[0] >= 28) {` here as the https://developer.chrome.com/blog/private-prefetch-proxy has been added to Chrome 103 for Android, and for other cases without version we can also assume Chrome version >= 28.

Reply:
I've refactored the code so it works the same way but is better readable at this place here.